pubnub-kotlin/pubnub-kotlin-test/src/commonMain/kotlin/com.pubnub.test/BaseIntegrationTest.kt

```diff
@@ -213,9 +212,11 @@ class PubNubTest(
                             return@createStatusListener
                         }
                         if ((pnStatus.category == PNStatusCategory.PNDisconnectedCategory || pnStatus.category == PNStatusCategory.PNSubscriptionChanged) &&
-                            (pnStatus.affectedChannels.containsAll(channels) && pnStatus.affectedChannelGroups.containsAll(
-                                channelGroups
-                            ))
+                            (
```

Comment:
Side note. Is this ok that event engine is disabled for JS ?

Reply:
yeah it's on purpose, there's some weird bug in some tests with EE enabled. I'm aware of it and trying to 1) fix the tests 2) find out if it's an error that can affect real users (haven't been able to reproduce outside of the test so far)

in the meantime ee can be false in tests